cohort_4/playbook/ACTIVITIES.md

```diff
@@ -104,6 +104,7 @@
     - [Announcing to the Whole Group](#announcing-to-the-whole-group)
     - [Sharing Announcements Individually](#sharing-announcements-individually)
     - [Slack Announcements](#slack-announcements)
+- [Additional Resources](#additional-resources)
```

Comment:
@ssun098 This is what is causing the tests to fail. There doesn't seem to be an `Additional Resources` heading.